go/vt/vttablet/tabletserver/planbuilder/ddl.go

```diff
@@ -39,17 +39,17 @@ func DDLParse(sql string) (plan *DDLPlan) {
 		return &DDLPlan{Action: ""}
 	}
 	return &DDLPlan{
-		Action:    stmt.Action,
-		TableName: stmt.Table,
-		NewName:   stmt.NewName,
+		Action:     stmt.Action,
+		TableNames: stmt.Tables,
+		NewName:    stmt.NewName,
 	}
 }
 
 func analyzeDDL(ddl *sqlparser.DDL, tables map[string]*schema.Table) *Plan {
 	// TODO(sougou): Add support for sequences.
 	plan := &Plan{
 		PlanID:  PlanDDL,
-		Table:   tables[ddl.Table.Name.String()],
+		Table:   tables[ddl.GetFirstTableName()],
```

Comment:
This part is tough, but the right thing to do is to change Table to Tables here, and iterate on all the tables wherever DDLs are handled. Without this, vttablet will not reload the schema for the rest of the tables.

This change goes deep because the Table info is also used in ACLs, etc. For now, you can change those parts to just use the first entry.

However, doing this will help us eventually fix the issue where we leave this field empty if there are multiple tables in a select.